### PR TITLE
convert z-index to American

### PR DESCRIPTION
**Pull request by michaelficarra**
This matches the documentation in the README.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ module.exports = postcss.plugin('css-properly', () => {
       d.prop = d.prop
         .replace('colour', 'color')
         .replace('padding-ton', 'padding-top')
+        .replace('z-index', 'z-index')
```
Comment (jevakallio) [inline index.js:9]:
```suggestion

```

Comment (michaelficarra) [reply index.js:9]: Oh, that's cheeky innit. But is it better to be explicit here? Don't want to confuse the Americans with our British ingenuity. :gb:

Comment (jevakallio) [reply index.js:9]: Ummmmmmmm I don't know if I can trust your input on this

image

Comment (michaelficarra) [reply index.js:9]: Hey, what are you on about? Have you never met a British expat? My English blood runs as red as the Liverpool FC kit. God save the Queen.

Comment (jevakallio) [reply index.js]: Oh yeah. Name the best three Oasis songs that aren't Wonderwall? (There is a correct answer)

**Comment by jevakallio** [comment]
@michaelficarra Thank you for the contribution! I merged the test in order to verify we don't break the documented behaviour, but sake good order, I've omitted the unnecessary implementation.

Thank you!

**Pull request merged**